.github/workflows/copilot-setup-steps.yml

```diff
@@ -0,0 +1,42 @@
+name: "Copilot Setup Steps"
+
+# Automatically run the setup steps when they are changed to allow for easy validation, and
+# allow manual testing through the repository's "Actions" tab
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - .github/workflows/copilot-setup-steps.yml
+
+jobs:
+  # The job MUST be called `copilot-setup-steps` or it will not be picked up by Copilot.
+  copilot-setup-steps:
+    runs-on: ubuntu-latest
+
+    # Set the permissions to the lowest permissions possible needed for your steps.
+    # Copilot will be given its own token for its operations.
+    permissions:
+      # If you want to clone the repository as part of your setup steps, for example to install dependencies, you'll need the `contents: read` permission. If you don't clone the repository in your setup steps, Copilot will do this for you automatically after the steps complete.
+      contents: write  # TEST: `uv` wasn't installed after the workflow succeded, maybe this is needed?
+      pull-requests: write  # So copilot can add labels to the PR
+
+    # You can define any steps you want, and they will run before the agent starts.
+    # If you do not check out your code, Copilot will do this for you.
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
```

Comment:
can you pin specific shas here (and in the user `uses:`) like in the other workflows? I don't quite understand why zizmore didn't run for this PR 🤔

Reply:
what do you mean by "user uses:"? If you meant the `uv` uses:, that already uses the latest point release